cucumber-messages/messages.proto

```diff
@@ -196,9 +196,14 @@ message Pickle {
 message PickleStep {
   string text = 1;
   repeated Location locations = 2;
-  oneof argument {
-    PickleDocString doc_string = 3;
-    PickleTable data_table = 4;
+  reserved 3, 4;
```

Comment:
I appreciate the use of `reserved` to ensure backwards compatibility. I think we can defer using it until we have made any releases that *actually* consume the message protocol. I think we can remove this for now.

Reply:
Hi @charlierudolph - sorry about the radio silence. I'm back to working on Cucumber full time as of today, so let's move this forward!

Reply:
On second thought, I think it's best to keep those `reserved` fields even though we haven't released anything yet. It's good to get into the habit!